electron/electron.ts

```diff
@@ -71,7 +82,10 @@ function createMenu() {
 
 app.whenReady().then(() => {
   createWindow();
```

Comment:
Here we could return the emitter and therefore we wouldn't have to do the `null` check, reassign after removing listeners, or even have the global reference.

Reply:
Great recommendation, thank you!